package.json

```
@@ -43,7 +43,8 @@
       "semver": "^7.5.4",
       "@types/react": "18.0.27",
       "goober": "2.1.9",
-      "react-hot-toast": "2.2.0"
+      "react-hot-toast": "2.2.0",
+      "postcss": "8.4.31"
```

I don't see why we should restrict minor/patch updates:
```suggestion
      "postcss": "^8.4.31"
```